### PR TITLE
New package: voltron-0.1.7

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/python3-Flask-RESTful/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-Flask-RESTful'
+pkgname=python3-Flask-RESTful
+version=0.2.12
+revision=1
+noarch=yes
+wrksrc="flask-restful-${version}"
+build_style=python3-module
```
[the remark this answers is not in the document]
Comment (abenson) [reply srcpkgs/python3-Flask-RESTful/template]: Fixed that and the other issues.

**Comment by abenson** [comment]
> +1 for only adding python3 stuff

Yeah, no point in adding a python3 version of the tool as well, as none of the dependencies are needed by anything python2 yet. Might as well just look foward only.

File: srcpkgs/voltron/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'voltron'
+pkgname=voltron
+version=0.1.7
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="voltron"
+hostmakedepends="python3-setuptools"
+depends="python3-blessed python3-Flask python3-Pygments python3-pysigset
+ python3-requests python3-scruffy"
```
Comment (maxice8) [inline srcpkgs/voltron/template]: is missing 

python3-requests-unixsocket
python3-Flask-RESTful

File: srcpkgs/python3-requests-unixsocket/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-requests-unixsocket'
+pkgname=python3-requests-unixsocket
+version=0.1.5
+revision=1
+noarch=yes
+wrksrc="requests-unixsocket-${version}"
+build_style=python3-module
+pycompile_module="requests_unixsocket"
+hostmakedepends="python3-setuptools python3-pbr"
+depends="python3-requests"
```
Comment (maxice8) [inline srcpkgs/python3-requests-unixsocket/template]: might also be a good idea to python3-urllib3 here, as it is implicity brought by python3-requests

**Pull request closed without merging**